.github/stale.yml

```diff
@@ -0,0 +1,51 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 14
```

Comment:
Maybe a month / 28 days gives people the chance to action the stale label. People tend to go away on holiday for 2 weeks in the summer, if it were my change and it happened days before / during a holiday I wouldn't action until my return.